modules/nw-sriov-configuring-device.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple-networks/configuring-sr-iov.adoc
+
+[id="configuring-sr-iov-devices_{context}"]
+= Configuring SR-IOV network devices
+
+The SR-IOV Network Operator adds the `SriovNetworkNodePolicy.sriovnetwork.openshift.io` Custom Resource Definition (CRD) to {product-title}.
+You can specify the SR-IOV network device and the configuration of the SR-IOV CNI plug-in by creating a SriovNetworkNodePolicy Custom Resource (CR).
```

Comment:
The configuration of SR-IOV CNI plug-in is defined via `SriovNetwork` CRD.

perhaps change to `You can configure the SR-IOV network device by creating a SriovNetworkNodePolicy Custom Resource (CR)`